src/pages/app-development/app-builder-partnership-essentials.md

```diff
@@ -1,53 +1,90 @@
 
-### Sandbox access
+#### Step 3: Request Sandbox access
 
-Once you are a Silver+ member and have set up your TPP account, you will need access to the following environments:
+After your Silver membership is processed, you can request sandbox access, which will be provisioned within 5 days of your request:
+
+- **App Builder Sandbox** - automatically associated with a TPP organization (for newly provisioned TPP organizations). Older organizations without permission should request access to all three sandboxes (App Builder, PaaS and ACCS/SaaS) through [TPP Support](https://partners.adobe.com/ec/cform/sandbox) on the same ticket (if needed).
+
+- **Commerce PaaS Sandbox** – Account Managers should request both Composer entitlements for Adobe Commerce EE/B2B Access and PaaS sandbox through [TPP Support](https://partners.adobe.com/ec/cform/sandbox).
+
+- **Commerce SaaS Sandbox** - File a ticket to [TPP Support](https://partners.adobe.com/ec/cform/sandbox). Once received, you can add additional users that will need sandbox access in your Admin Console. See Step 6 to manually assign users as Admins or Developers (check step 6).
+
+<InlineAlert variant="info" slots="text"/>
 
-- **App Builder Sandbox** - automatically associated with a TPP organization (for newly provisioned TPP organizations). Older organizations without permission should request a sandbox through [TPP Support](https://partners.adobe.com/ec/cform/sandbox) and request access to the following Commerce Sandboxes on the same ticket (if needed).
+You can create a ticket to check the status of your organization, or sandbox enablement, specifying the name of your assigned TPP organization.
 
-- **Commerce PaaS Sandbox** – Account Managers should request Composer entitlements for Adobe Commerce EE/B2B Access through [TPP Support](https://partners.adobe.com/ec/cform/sandbox).
+#### Step 4: Log into your IMS organization
 
-- **Commerce SaaS Sandbox** - access follows the same process as PaaS. Please file a ticket to [TPP Support](https://partners.adobe.com/ec/cform/sandbox).
+Verify you are logged into your IMS organization by checking the top-right corner of Adobe Experience Cloud or Developer Console. If it displays your organization name, you're ready to use App Builder. If it shows an email address, you're signed into a personal account instead of your IMS organization.
+
+#### Step 5: Create Project from template
+
+1. Navigate to the Developer Console to access App Builder.
+
+1. Select **Create Project from template**.
+
+   ![Create project from template](../_images/developer_console_template.png)
+
+1. Follow steps described in the [Create your First App Builder Application](https://developer.adobe.com/app-builder/docs/get_started/app_builder_get_started/first-app) topic.
```

Comment:
It might be better to point the reader to the Integration and Checkout starter kits (in this repo)